js/renovation/button.tsx

```diff
@@ -208,18 +208,18 @@ export default class Button extends JSXComponent<ButtonProps> {
     useSubmitBehavior && this.submitInputRef.click();
   }
 
-  onWidgetKeyDown(event: Event, options) {
+  onWidgetKeyDown(options) {
     const { onKeyDown } = this.props;
-    const { keyName, which } = options;
+    const { originalEvent, keyName, which } = options;
```

Comment:
Why we use the name `originalEvent`? Can we call it just `event`?

Reply:
The `originalEvent` came from old component. But we can rename this.

Reply:
In fact, `options` - is `event` argument in `onKeyDown(event)`. I think for this reason we have `originalEvent` field.

Reply:
I see, but in the current context, we don't have any `event` variables and can make our code more readable.